test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -115,8 +118,17 @@ def hook(context):
 
 @OnScenarioEnd
 def hook(context):
+    global previousFailResultCount
+    global previousErrorResultCount
     # capture screenshot if there is error in the scenario execution, and if the test is being run in CI
-    if test.resultCount("errors") > 0 and os.getenv('CI'):
+    if (
+        (test.resultCount("fails") - previousFailResultCount) > 0
+        or (test.resultCount("errors") - previousErrorResultCount) > 0
+        and os.getenv('CI')
+    ):
+        previousFailResultCount = test.resultCount("fails")
+        previousErrorResultCount = test.resultCount("errors")
+
```

Comment:
Please, add some commit for this workaround.
It would be useful in the future to know why the global vars were needed.